docs/ext/rfc.py

```diff
@@ -39,11 +40,10 @@ def _process_line(line):
     section = m.group(2)
 
     template = (
-        '`RFC {rfc}, Section {section} '
-        '<https://tools.ietf.org/html/rfc{rfc}#section-{section}>`_'
+        '`RFC {rfc}, Section {section} ' '<{ietf_docs}/rfc{rfc}#section-{section}>`__'
```

Comment:
This template is used late as a template, I don't think fstring fits in that case. I'll join these, it was ruff that put them together when I shortened the lines.

Reply:
Ah, it seems you are right, the template is only used once, so it could well be an fstring. But let's keep the increment minimal for now. I applied your suggestion.